collectors/ebpf.plugin/README.md

```diff
@@ -116,8 +192,33 @@ processes](https://en.wikipedia.org/wiki/Zombie_process).
 
 #### Task error
 
-The functions responsible for ending tasks do not return values, so this chart contains information about failures on
-process and thread creation.
+The functions responsible for ending tasks do not return values, so this chart contains information about failures on 
+process and thread creation only.
+
+#### TCP functions
+
+This chart demonstrates calls to functions `tcp_sendmsg`, `tcp_cleanup_rbuf` and `tcp_close`; these functions are
```

Comment:
```suggestion
This chart demonstrates calls to functions `tcp_sendmsg`, `tcp_cleanup_rbuf`, and `tcp_close`; these functions are
```

Reply:
Actually a comma before the "and" in a list is not necessary. At least that's what my 9th grade English teacher used to insist 😄

Reply:
Alright well seems like it's "optional": https://www.grammarly.com/blog/comma-before-and/

Reply:
https://learn.netdata.cloud/contribute/style-guide#oxford-comma

Reply:
@UmanShahzad I also heard this a lot not only in English classes, but for Latin languages too. :smile: